source/_integrations/doods.markdown

```diff
@@ -11,8 +11,16 @@ ha_domain: doods
 The `doods` image processing integration allows you to detect and recognize objects in a camera image using [DOODS](https://github.com/snowzach/doods/). The state of the entity is the number of objects detected and recognized objects are listed in the `summary` attribute along with quantity. The `matches` attribute provides the confidence `score` for recognition and the bounding `box` of the object for each detection category.
 
 ## Setup
-
-You need to have DOODS running somewhere. It's easiest to run as a [Docker](https://hub.docker.com/r/snowzach/doods) container.
+The DOODS software needs to be running, before this integration can be used. Options to run the DOODS software:
+### Run as Home Assistant add-on
+This is the easiest way when you are running supervised Home Assistant and want to run the DOODS software on the same machine as Home Assistant
+1. Add snowzach repository (https://github.com/snowzach/hassio-addons) to the add-on store (see [add-on documentation](https://www.home-assistant.io/hassio/installing_third_party_addons/) for more information )
+2. Go to the add-on store, you should now see the new repository
+3. Install DOODS 
+4. Go to dashboard, select DOODS and start DOODS
+
+### Run as a Docker container
+https://hub.docker.com/r/snowzach/doods
```

Comment:
We should not provide a manual, that is for the upstream project to document. Not us.

Please adjust this to link to the add-on and docker containers, preferably with just a link to their installation instructions.

Reply:
Thanks for your feedback, I've reworked this and will submit pullrequest